tests/bluetooth/tester/src/audio/btp_bap_broadcast.c

```diff
@@ -484,13 +487,22 @@ uint8_t btp_bap_broadcast_source_stop(const void *cmd, uint16_t cmd_len,
 
 	LOG_DBG("");
 
+	k_sem_reset(&sem_bis_stopped);
+
 	err = bt_bap_broadcast_source_stop(source->bap_broadcast);
 	if (err != 0) {
 		LOG_DBG("Unable to stop broadcast source: %d", err);
 
 		return BTP_STATUS_FAILED;
 	}
 
+	err = k_sem_take(&sem_bis_stopped, K_MSEC(2000));
+	if (err != 0) {
+		LOG_DBG("Timed out waiting to stop BIS stream");
+
+		return BTP_STATUS_FAILED;
+	}
```

Comment:
`stream_stopped` that gives `sem_bis_stopped` may be called multiple times. 

You should take the same number of semaphores as we have streams.

Reply:
This should now be resolved, given bt_bap_broadcast_source_stop stops broadcast source streams only.